tensorflow/lite/micro/kernels/depthwise_conv_test.cc

```diff
@@ -579,7 +579,7 @@ TF_LITE_MICRO_TEST(FilterDimsNotMatchingAffineQuantization) {
   int8_t input_quantized[input_size];
   int8_t filter_quantized[filter_size];
   int32_t bias_quantized[bias_size];
-  int8_t golden_quantized[output_size];
+  int8_t golden_quantized[output_size] = {0};
```

Comment:
nit: let's have this be
```
int8_t golden_quantized[output_size] = {};
```
since 
```
int8_t golden_quantized[output_size] = {1};
```
will only initialize the first element to 1 and default initialize everything else.

Reply:
Done